client/testserver/src/main/scala/sttp/tapir/client/tests/HttpServer.scala

```diff
@@ -133,8 +133,27 @@ class HttpServer(port: Port) {
           val e = q.enqueue
           WebSocketBuilder[IO].build(d, e)
         }
+
+    case r @ GET -> Root / "api" / "organization" =>
```

Comment:
maybe it would be good to change the `api` into something that indicates where this is used - here, for content-negotiation tests. So maybe `/content-negotiation/organization`?